app/main.py

```diff
@@ -1,3 +1,15 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    stored_results = dict()
```

Comment:
it is more tipical to make like this `stored_results = {}`, but it's up to u

Reply:
> it is more tipical to make like this `stored_results = {}`, but it's up to u

I heard that in previous versions of python {} was to assign a set collection. Don't know that they changed it

Reply:
it seems to me, that it can be the syntax of 2.x python

Reply:
> it seems to me, that it can be the syntax of 2.x python

Thank you for the information! I will use {} instead of dict() in my next solutions